cocoa/scheduler.py

```diff
@@ -1,24 +1,44 @@
+import asyncio
 from datetime import datetime, timedelta
 from notifier import NotifierService
 from model import ModelService
 from discord import Client, TextChannel, User
 
 
+async def get_time(message: str, channel: TextChannel, client: Client):
+    await channel.send(message)
+    try:
+        response = await client.wait_for('message', timeout=120)
+    except asyncio.TimeoutError:
+        return await channel.send('Sorry, you took too long to make a choice.')
```

Comment:
Can anyone of you think of a more user-friendly way to get the start date and end date of a meeting?